app/server/api/user_api.py

```diff
@@ -126,6 +126,7 @@ def post(self, user_id):
 
         post_data = request.get_json()
 
+        # TODO(admin_create): pass in current user
         response_object, response_code = UserUtils.proccess_create_or_modify_user_request(
```

Comment:
are we able to pull info about the user making the API call? or do we just have their token to validate they have the right permissions?

Reply:
we need it for `enrolled_by`

Reply:
oh yep we already do this! There's a column on the base model class called "authorising_user_id"  which is by default set to the user id of the user making the api call, as determined by their auth token:

```
    authorising_user_id = db.Column(db.Integer, default=get_authorising_user_id)
.
.
.
def get_authorising_user_id():
    if hasattr(g, 'user'):
        return g.user.id
    elif hasattr(g, 'authorising_user_id'):
        return g.authorising_user_id
    else:
        return None

```

so u can either just use that as `enrolled_by`, or set it using a similar process, or run `enrolled_by` as a hybrid property.

Reply:
huh, i see that authorising_user_id is on every model, but i don't see where it gets set. where is that happening?

Reply:
in the column defintion itself:
`    authorising_user_id = db.Column(db.Integer, default=get_authorising_user_id)`

The default part calls the function

Reply:
coool!